### PR TITLE
Docs: Guide: Get your Tequila API key

### LINKED ISSUE #545: Docs: Guide: Get your Tequila API key
Prepare guide how to get Tequila API key.
should be live in: http://localhost:1337/margarita/docs/guide-tequila-api-key

### PR DESCRIPTION
**Pull request by FilipMessa**
close #545

### CODE REVIEW

File: docs/guide-tequila-api-key.md [hunk 1]
```diff
@@ -1,6 +1,20 @@
...
+. You can register and log in on the tequila.kiwi.com page.
+
+After successfully signing in to your account, you can find out what **Tequila** is about, how to work directly with the **Tequila API**.
+
+To get your **API key** for the GraphQL got into the My applications page.
+Create the new app, by clicking on the `add application` button.
+
+Then you should choose your partnership the **<span>Kiwi</span>.com Affiliate Program** or **Book with <span>Kiwi</span>.com**. More information about partnership types, you will find there.
+
+After choosing your partnership, you need to select the product type, our GraphQL proxy support only the _Search_ and _Search & Book_ features, but the _Multicity_ and _Nomad_ are on our roadmap and will come later.
```
Comment (michalsanger) [inline docs/guide-tequila-api-key.md]: After choosing your partnership, you need to select the product type. Currently, the GraphQL proxy supports only _Search_ and _Search & Book_ features. The _Multicity_ and _Nomad_ search will be supported later.

File: docs/guide-tequila-api-key.md [hunk 2]
```diff
@@ -1,6 +1,20 @@
...
+
+When you have been created application you will be redirected to the application detail page where you can grab your key from the Detail section.
```
Comment (michalsanger) [inline docs/guide-tequila-api-key.md]: Once you successfully created the application you will be redirected to the application detail page where you can grab your **API key** from the Detail section.

File: docs/guide-tequila-api-key.md [hunk 3]
```diff
@@ -1,6 +1,20 @@
...
+
+To get your **API key** you will need an account at the [Tequila](tequila.kiwi.com)
+. You can register and log in on the tequila.kiwi.com page.
+
+After successfully signing in to your account, you can find out what **Tequila** is about, how to work directly with the **Tequila API**.
+
+To get your **API key** for the GraphQL got into the My applications page.
+Create the new app, by clicking on the `add application` button.
+
+Then you should choose your partnership the **<span>Kiwi</span>.com Affiliate Program** or **Book with <span>Kiwi</span>.com**. More information about partnership types, you will find there.
```
Comment (michalsanger) [inline docs/guide-tequila-api-key.md]: Why the `<span>` tags?

> More information about partnership types, you will find there.
Where? I'd probably remove this sentence if the info is available on the page.

File: docs/guide-tequila-api-key.md [hunk 4]
```diff
@@ -1,6 +1,20 @@
...
-_This page is a stub._
+For successful run our the GraphQL proxy server, you need to has set the Tequila **API key** to `API_KEY` environment variable.
+
+To get your **API key** you will need an account at the [Tequila](tequila.kiwi.com)
+. You can register and log in on the tequila.kiwi.com page.
+
+After successfully signing in to your account, you can find out what **Tequila** is about, how to work directly with the **Tequila API**.
+
+To get your **API key** for the GraphQL got into the My applications page.
+Create the new app, by clicking on the `add application` button.
```
Comment (michalsanger) [inline docs/guide-tequila-api-key.md]: To get your **API key** go to the My Applications page and create the new app, by clicking on the `Add Application` button.

> go to the My Applications page

Can the "My Application page" be a link?

File: docs/guide-tequila-api-key.md [hunk 5]
```diff
@@ -1,6 +1,20 @@
...
+sidebar_label: Guide: Get your Tequila API key
 ---
 
-_This page is a stub._
+For successful run our the GraphQL proxy server, you need to has set the Tequila **API key** to `API_KEY` environment variable.
+
+To get your **API key** you will need an account at the [Tequila](tequila.kiwi.com)
+. You can register and log in on the tequila.kiwi.com page.
+
+After successfully signing in to your account, you can find out what **Tequila** is about, how to work directly with the **Tequila API**.
```
Comment (michalsanger) [inline docs/guide-tequila-api-key.md]: After successfully signing in to your account, you can read more information about **Tequila** and also explore the standard REST API.

File: docs/guide-tequila-api-key.md [hunk 6]
```diff
@@ -1,6 +1,20 @@
 ---
-title: "Guide: Get your Tequila API key"
-sidebar_label: "Guide: Get your Tequila API key"
+title: Guide: Get your Tequila API key
+sidebar_label: Guide: Get your Tequila API key
 ---
 
-_This page is a stub._
+For successful run our the GraphQL proxy server, you need to has set the Tequila **API key** to `API_KEY` environment variable.
```
Comment (michalsanger) [inline docs/guide-tequila-api-key.md]: Margarita fetches data from GraphQL server. This server is just a proxy of officially supported Tequila REST API. The REST API requires an API key to be included in every call. GraphQL server loads the value from an environment variable named `API_KEY`.

File: docs/guide-tequila-api-key.md [hunk 7]
```diff
@@ -1,6 +1,20 @@
...
+
+To get your **API key** you will need an account at the [Tequila](tequila.kiwi.com)
+. You can register and log in on the tequila.kiwi.com page.
```
Comment (michalsanger) [inline docs/guide-tequila-api-key.md]: To get your **API key** you will need to create an account on the [Tequila](tequila.kiwi.com) website. The registration is free and gives you also access to other features of Tequila platform.

**Comment by michalsanger** [comment]
I don't feel like I'm the best one for proofreading, feel free to find better english skilled colleague :-)

**Comment by FilipMessa** [comment]
@michalsanger thanks 👍

File: docs/guide-tequila-api-key.md (hunk 3 above)
Comment (FilipMessa) [reply docs/guide-tequila-api-key.md]: Because without this "hack", it behaves like a link

File: docs/guide-tequila-api-key.md (hunk 1 above)
Comment (FilipMessa) [reply docs/guide-tequila-api-key.md]: done

File: docs/guide-tequila-api-key.md (hunk 2 above)
Comment (FilipMessa) [reply docs/guide-tequila-api-key.md]: done

File: docs/guide-tequila-api-key.md (hunk 3 above)
Comment (FilipMessa) [reply docs/guide-tequila-api-key.md]: done

File: docs/guide-tequila-api-key.md (hunk 5 above)
Comment (FilipMessa) [reply docs/guide-tequila-api-key.md]: done

File: docs/guide-tequila-api-key.md (hunk 4 above)
Comment (FilipMessa) [reply docs/guide-tequila-api-key.md]: done

File: docs/guide-tequila-api-key.md (hunk 7 above)
Comment (FilipMessa) [reply docs/guide-tequila-api-key.md]: done

File: docs/guide-tequila-api-key.md (hunk 6 above)
Comment (FilipMessa) [reply docs/guide-tequila-api-key.md]: done

File: docs/guide-tequila-api-key.md [hunk 8]
```diff
@@ -1,6 +1,19 @@
 ---
-title: "Guide: Get your Tequila API key"
-sidebar_label: "Guide: Get your Tequila API key"
+title: Guide: Get your Tequila API key
+sidebar_label: Guide: Get your Tequila API key
 ---
 
-_This page is a stub._
+Margarita fetches data from GraphQL server. This server is just a proxy of officially supported Tequila REST API. The REST API requires an **API key** to be included in every call. GraphQL server loads the value from an environment variable named `API_KEY`.
```
Comment (RobinCsl) [inline docs/guide-tequila-api-key.md]:
```suggestion
Margarita fetches data from the GraphQL server under `apps/graphql`. This server is just a proxy of officially supported Tequila REST API. The REST API requires an **API key** to be included in every call. The GraphQL server loads the value from an environment variable named `API_KEY`.
```

File: docs/guide-tequila-api-key.md [hunk 9]
```diff
@@ -1,6 +1,19 @@
...
+
+To get your **API key** you will need to create an account on the [Tequila](tequila.kiwi.com) website. The registration is free and gives you also access to other features of Tequila platform.
```
Comment (RobinCsl) [inline docs/guide-tequila-api-key.md]:
```suggestion
To get your **API key** you will need to create an account on the [Tequila](https://tequila.kiwi.com) website. The registration is free and gives you also access to other features of Tequila platform.
```

File: docs/guide-tequila-api-key.md [hunk 10]
```diff
@@ -1,6 +1,19 @@
...
+
+After successfully signing in to your account, you can read more information about **Tequila** and also explore the standard REST API.
+
+To get your **API key** for the GraphQL got into the [My applications](https://tequila.kiwi.com/portal/my-applications) page.
```
Comment (RobinCsl) [inline docs/guide-tequila-api-key.md]:
```suggestion
To get your **API key** for the GraphQL server, go to the [My applications](https://tequila.kiwi.com/portal/my-applications) page.
```

File: docs/guide-tequila-api-key.md [hunk 11]
```diff
@@ -1,6 +1,19 @@
...
+Create the new app, by clicking on the `add application` button.
```
Comment (RobinCsl) [inline docs/guide-tequila-api-key.md]:
```suggestion
Create a new app by clicking on the `add application` button.
```

File: docs/guide-tequila-api-key.md [hunk 12]
```diff
@@ -1,6 +1,19 @@
...
+
+Then you should choose your partnership the **<span>Kiwi</span>.com Affiliate Program** or **Book with <span>Kiwi</span>.com**. More information about partnership types, you will find there.
```
Comment (RobinCsl) [inline docs/guide-tequila-api-key.md]:
```suggestion
You should then choose the type of partnership you want with <span>Kiwi</span>.com: either **<span>Kiwi</span>.com Affiliate Program** partnership or **Book with <span>Kiwi</span>.com**  partnership. A description of these partnerships is available during this step.
```

File: docs/guide-tequila-api-key.md [hunk 13]
```diff
@@ -1,6 +1,19 @@
...
+
+After choosing your partnership, you need to select the product type. Currently, the GraphQL proxy supports only _Search_ and _Search & Book_ features. The _Multicity_ and _Nomad_ search will be supported later.
```
Comment (RobinCsl) [inline docs/guide-tequila-api-key.md]:
```suggestion
After choosing your partnership, you need to select the product type. Currently, the GraphQL proxy supports only _Search_ and _Search & Book_ features. The _Multicity_ and _Nomad_ search will be supported at a later date.
```

**Pull request merged**